tensorflow_model_optimization/g3doc/guide/pruning/index.md

```diff
@@ -51,12 +65,14 @@ It is on our roadmap to add support in the following areas:
       <th>Model</th>
       <th>Non-sparse Top-1 Accuracy </th>
       <th>Sparse Accuracy </th>
+      <th>Sparsity 2 by 4</th>
       <th>Sparsity </th>
     </tr>
     <tr>
       <td rowspan=3>InceptionV3</td>
       <td rowspan=3>78.1%</td>
       <td>78.0%</td>
+      <td>75.8%</td>
```

Comment:
The current table is hard to understand. I think we can do either

Sparse accuracy | Sparsity
77%.                    | 50%
69%                    | 2 by 4

or 
Ramdom sparse accuracy | Random sparsity | Structured sparse accuracy | Structured sparsity
77%.                    | 50%.      |   69%                    | 2 by 4


And just FYI, the original numbers for inceptionv3 in this table is from the paper linked below - https://arxiv.org/pdf/1710.01878.pdf